types/types.go

```diff
@@ -334,3 +334,37 @@ type AccessLog struct {
 	FilePath string `json:"file,omitempty" description:"Access log file path"`
 	Format   string `json:"format,omitempty" description:"Access log format: json | common"`
 }
+
+// HasCustomHeadersDefined checks to see if any of the custom header elements have been set
+func (h *Headers) HasCustomHeadersDefined() bool {
+	if len(h.CustomResponseHeaders) == 0 &&
```

Comment:
This can be simplify:
```golang
return !(len(h.CustomResponseHeaders) == 0 &&
		len(h.CustomRequestHeaders) == 0)
```
or 
```golang
return len(h.CustomResponseHeaders) != 0 ||
		len(h.CustomRequestHeaders) != 0
```